### DISCUSSION TITLE
Idk how use this project

### DISCUSSION THREAD

**Discussion by RudyRyc2000** [psu-de/MineSharp#27] [Q&A]
Hi 🖐
I download from nuget package MineSharp but when i use "using" to load namespace this namespace doesn't exist.

**Comment by psu-de** [comment]
Hey and welcome 🖐,
I have only published MineSharp.Core once as a test a year ago, but that's really old.
For now I'd suggest to clone the Repository, create a project and reference MineSharp.Bot. Then you can use it just like in the example in the Readme. 
Currently I'm working on Physics, after that, the framework is pretty much complete and I'll start publishing NuGet packages.
